### ISSUE TITLE
Is there a way to connect two 86Box instances together via virtual LAN?

### ISSUE THREAD

**Issue by raduzaharia-medium** [86Box/86Box#3371]
I would like to start two 86Box VMs in parallel and link them together in a virtual LAN. Is this possible?

**Comment by daviunic** [comment]
Transfered to the emulator repo since this is unrelated to the Manager.

**Comment by OBattler** [comment]
Closing this as being a duplicate of this already closed issue; https://github.com/86Box/86Box/issues/2252 .
